lib/php/lib/Thrift/Transport/TSocket.php

```diff
@@ -255,6 +255,16 @@ public function close()
   }
 
   /**
+   * Close the socket and try to
+   */
+  public function reconnect()
```

Comment:
I don't see what this has to do with seqId changes and it is incorrect.  You would want to call the TSocket::close() and not fclose here.